src/ts/components/seekbar.ts

```diff
@@ -164,12 +213,20 @@ export class SeekBar extends Component<SeekBarConfig> {
         } else {
           let playbackPositionPercentage = 100 - (100 / player.getMaxTimeShift() * player.getTimeShift());
           this.setPlaybackPosition(playbackPositionPercentage);
+          this.setAriaSliderMinMax(player.getMaxTimeShift().toString(), '0');
+        }
+
+        if (this.isUiShown) {
+          const timeshiftValue = Math.ceil(this.player.getTimeShift()).toString();
+          this.getDomElement().attr('aria-valuenow', timeshiftValue);
+          this.getDomElement().attr('aria-valuetext', `${i18n.performLocalization(i18n.getLocalizer('seekBar.timeshift'))} ${i18n.performLocalization(i18n.getLocalizer('seekBar.value'))}: ${timeshiftValue}`);
         }
```

Comment:
Why are these values only set then the UI is shown? Are they ever reset when the UI is hidden? This should be implemented in a reactive way, directly depending on the events. Setting a flag in the events and then reacting at some undefined later point in time leads to ugly edge cases.

Reply:
They stay as they are when the UI is hidden. We don't want to update the attributes because the user would never hear the video if it keeps updating that is why we stop updating until the UI is shown . We are listening to the `uimanager.onControlsShow`

Reply:
> They stay as they are when the UI is hidden.

So you deliberately want to keep values which are then wrong and misrepresent the player state?

Reply:
Correct because otherwise it would be unintuitive for the user

Reply:
Keeping outdated values is more intuitive than resetting an outdated state?

Reply:
But to what would the values reset to? They get updated when the UI is shown again. If we change the values in any way when the UI is hidden the user will hear that. The user doesn't need that info as it will make noise for hime while he is listening to the video

Reply:
> But to what would the values reset to?

The state they were is before being populated first. When I look into the DOM I don't want to see irrelevant outdated values.

Will the screenreader talk about values being removed?

Reply:
Yes it reads content that is not contained inside the element. I checked out other players for various sources and all of them use the same or similar system as I implemented here